nowastecrawler/nowastecrawler/inserting_to_mongo.py

```diff
@@ -0,0 +1,39 @@
+import pymongo  # biblioteka mongo
+import pandas as pd  # python -m pip install pandas
+import json
+
+DATABASE_HOST = "localhost"
```

Comment:
Can we read it from the same source as the server? Look for example at https://pypi.org/project/python-dotenv/ as a simple solution.